ADO.NET/OAuthSampleApp/app.config

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+	<!--
+      ConnectionStrings defines how the SampleApp will connect to the server.  Change this to be specific to your environment.
+    -->
+	<connectionStrings>
+		<add name="ConnectionString" connectionString="Host=localhost;Port=5433;Database=VMart;User=dbadmin;" providerName="Vertica.Data" />

```

Comment:
The user is a required part of the connection string in ADO.NET. I think there is a Jira out there somewhere to change this.

Reply:
What do you think about defining two connection strings?  One could be used to set up the database and the other for OAuth.  For the OAuth connection, you could add a comment that states that the driver requires a User value, but it is ignored.   I want to make it very clear that when connecting with OAuth, the user value is not used by the driver, but it can also not be empty.

Reply:
This might also clean up the code that modifies the connection string when calling SetUp.

Reply:
The only thing that needs to change for the connection string is the user and I don't like having the 'demo user' in two different parts of the configuration. I will make the comments more clear, though.